projects/plugins/jetpack/extensions/blocks/dialogue/components/participants-control.js

```diff
@@ -1,45 +1,40 @@
 /**
  * WordPress dependencies
  */
-import {
-	DropdownMenu,
-	MenuGroup,
-	MenuItem,
-	SelectControl,
-} from '@wordpress/components';
+import { DropdownMenu, MenuGroup, MenuItem, SelectControl } from '@wordpress/components';
 import { check } from '@wordpress/icons';
 import { __ } from '@wordpress/i18n';
 
 function ParticipantsMenu( { participants, className, onSelect, participantSlug, onClose } ) {
 	return (
 		<MenuGroup className={ `${ className }__participants-selector` }>
-			{ participants.map( ( { participant, participantSlug: slug } ) => (
+			{ participants.map( ( { label, slug } ) => (
```

Comment:
Changes in this file are related to adapting to the new participant attribute shape in the conversation and dialogue blocks.